### PR TITLE
Added a tracker hud to gang tags

### PR DESCRIPTION
**Pull request by TobleroneSwordfish**
[QOL] [GAMEMODES]
## About the PR 
Makes attempting to tag an area that already has a tag add a temporary tracker component to your HUD pointing to the offending tag. Lasts 3 seconds.

## Why's this needed? 
Cuts the time spent searching for annoyingly hidden tags down to hopefully less than 75% of every gang round.
PRing because it's late and I don't want to think about balance implications right now.

## Changelog 

```changelog
(u)LeahTheTech
(+)Trying to gang tag a room containing a rival tag will now point you in the direction of the offending tag.
```

### CODE REVIEW

**Comment by nefarious6th** [comment]
i feel like part of tagging inconspicuous places is that it should be rewarded by being harder to find; why bother trying to hide the location if someone will just get the ability to see + clean the tag immediately

**Comment by BatElite** [comment]
I like the simpler solution of just layering the tags over stuff

hunting for the bastards isn't exactly engaging

**Comment by ZeWaka** [comment]
@TobleroneSwordfish

File: code/datums/gamemodes/gangwar.dm [hunk 1]
```diff
@@ -651,6 +651,14 @@
 			boutput(user, "<span class='alert'>You don't have the dexterity to spray paint a gang tag!</span>")
 		if(getarea.gang_owners && getarea.gang_owners != user.mind.gang && !turftarget.tagged)
 			boutput(user, "<span class='alert'>[getarea.gang_owners.gang_name] own this area! You must paint over their tag to capture it!</span>")
+			if (user.GetComponent(/datum/component/tracker_hud))
+				return
+			for (var/turf/T in getarea)
+				if (T.tagged)
```
Comment (ZeWaka) [inline code/datums/gamemodes/gangwar.dm:657]: owie

Comment (TobleroneSwordfish) [reply code/datums/gamemodes/gangwar.dm:657]: guh

**Pull request merged**